packages/commercetools/theme/helpers/checkout/compareAddresses.ts

```diff
@@ -0,0 +1,12 @@
+export const compareAddresses = (savedAddress, currentAddress) => {
+  return savedAddress.city === currentAddress.city &&
```

Comment:
Use [isEqual](https://lodash.com/docs/4.17.15#isEqual) from lodash to compare those two objects. Otherwise it will be required to update this function everytime properties change and it's prone to errors.

Reply:
It is good advice. But I found we do not need `compareAddresses`, at least for now :D